controls/radchartview3d/populating-data/data-binding.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Data Binding
+page_title: Data Binding
+description: Data Binding
+slug: radchartview3d-populating-data-databinding
+tags: axes,3d,chartview,data,binding,itemssource,valuebinding,
+published: True
+position: 3
+---
+
+# Data Binding
+
+This article explains how to bind a 3D chart series to a collection of business objects. To data bound the series you can use its __ItemsSource__ property. The property is of type __IEnumerable__ so you can use most collection types. 
+
+The chart will create a __DataPoint3D__ model for each business object in the ItemsSource collection. To map the properties of the business object to the DataPoint3D you can use the series value binding properties.
+
+* __XValueBinding__: The binding that will be used to fill the data point model XValue property.
+* __YValueBinding__: The binding that will be used to fill the data point model YValue property.
+* __ZValueBinding__: The binding that will be used to fill the data point model ZValue property.
+
+The properties are of type __DataPointBinding__ which is an abstract class that has two implementations - __PropertyNameDataPointBinding__ and __GenericDataPointBinding__.
+
+The __PropertyNameDataPointBinding__ excepts a path (name) to a property in the business object of the data point. The binding class uses a reflection and the property name to get the corresponding value and assing it to the data point model. This type of binding is used when you set the value binding properties in XAML.
+
+#### __[C#] Example 1: Using PropertyNameDataPointBinding__
+{{region cs-radchartview3d-populating-data-databinding-0}}
+	chartSeries3D.XValueBinding = new PropertyNameDataPointBinding("MyXValue");
+{{endregion}}
+
+#### __[XAML] Example 2: Setting PropertyNameDataPointBinding in XAML__
+{{region xaml-radchartview3d-populating-data-databinding-1}}
+	<telerik:BarSeries3D XValueBinding="MyXValue"/>
+{{endregion}}
+
+The __GenericDataPointBinding&lt;TElement, TResult&gt;__ expects a function that selects a value from the business object. Then it sets this value to the corresponding property of the data point model. This type of binding can be used only in code. 
+
+#### __[C#] Example 3: Using GenericDataPointBinding__
+{{region cs-radchartview3d-populating-data-databinding-2}}
+	chartSeries3D.XValueBinding = new GenericDataPointBinding<MyBusinessObject, double>()
+	{
+		ValueSelector = x => x.MyXValue
+	};
+{{endregion}}
+	
+In summary, to data bound a 3D chart series you will need to set 3 properties - __YValueBinding, XValueBinding, ZValueBinding and ItemsSource__.
```

Comment:
+In summary, to bind data to a 3D chart series, you will need to set 3 (aren't they 4? :)) properties - __YValueBinding, XValueBinding, ZValueBinding and ItemsSource__.